src/java.base/share/classes/sun/security/x509/AuthorityKeyIdentifierExtension.java

```diff
@@ -155,41 +155,25 @@ public AuthorityKeyIdentifierExtension(Boolean critical, Object value)
                                   "AuthorityKeyIdentifierExtension.");
         }
 
-        // Note that all the fields in AuthorityKeyIdentifier are defined as
-        // being OPTIONAL, i.e., there could be an empty SEQUENCE, resulting
-        // in val.data being null.
```

Comment:
Is this comment not valid anymore? Don't we need to check for (val.data != null) in the new code?